README.md

```diff
@@ -57,11 +57,12 @@ import AppStoreServerLibrary
 
 let bundleId = "com.example"
 let appleRootCAs = loadRootCAs() // Specific implementation may vary
+let appAppleId: String? = nil // For production environments, it's required
```

Comment:
This is an Int64?, also could you please use the phrase "appAppleId must be provided for the Production environment"